tests/conftest.py

```diff
@@ -186,27 +288,76 @@ def schema_adv(connection_test):
 
 
 @pytest.fixture
-def httpClient():
+def schema_ext(connection_test, stores_config, enable_filepath_feature):
+    schema = dj.Schema(
+        PREFIX + "_extern",
+        context=schema_external.LOCALS_EXTERNAL,
+        connection=connection_test,
+    )
+    dj.config["stores"] = stores_config
+    dj.config["cache"] = tempfile.mkdtemp()
+
+    schema(schema_external.Simple)
+    schema(schema_external.SimpleRemote)
+    schema(schema_external.Seed)
+    schema(schema_external.Dimension)
+    schema(schema_external.Image)
+    schema(schema_external.Attach)
+
+    # dj.errors._switch_filepath_types(True)
+    schema(schema_external.Filepath)
+    schema(schema_external.FilepathS3)
+    # dj.errors._switch_filepath_types(False)
```

Comment:
```suggestion
    schema(schema_external.Filepath)
    schema(schema_external.FilepathS3)
```
remove commented code to dodge dmitri aggro